google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -827,15 +820,13 @@ def _convert_iam_policy(self, iam_policy):
             iam_policy (object): IAM policy to store.
         """
         _, full_res_name = self._get_parent(iam_policy)
-        parent_type_name = self._parent_type_name(iam_policy)
+        parent_type_name = self._type_name(iam_policy)
```

Comment:
qq: another question here about the switch from `_parent_type_name()` to `_type_name()`

Reply:
An iam_policy has the same type_name as it's parent object, and we no longer track the parent_resource_type and parent_resource_id in the object.